Project 3/Task 1/Utility.py

```diff
@@ -18,8 +20,28 @@ def safeGetDirectory():
         except:
             print("Error getting directory")
 
-
-
+def getNValue():
+      while 1:
+        try:
+            option = input('Please enter an m value: ')
+            if option < 1:
+                print(str(option) + ' is not a valid selection. Please make a different selection.\n')
+            else:
+                return option
+        except:
+            print(str(option) + ' is not a valid input. Please try again.\n')
+
+def getContent(video):
+    frameData = list()
+    while(video.isOpened()):
+        ret, frame = video.read()
+        if ret:
+            yFrameValues = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
+            frameData.append(yFrameValues)
+        else:
+            break
+    # Possibly revisit to add max and min here
+    return frameData
```

Comment:
Moved `getNValue` and `getContent` to the utility file since other files will probably need this.